_rules/80f0bf.md

```diff
@@ -0,0 +1,112 @@
+---
+name: No auto-play audio
+test_type: composite
+
+description: |  
+  This rule checks that auto-play audio does not last for more than 3 seconds, or it has a control mechanism.
+
+success_criterion: 
+- 1.4.2 # Audio Control
+
+atomic_rules:
+- 95e043
+- aaa1bf
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>` `<video>` or `<source>` `src` file content with a `duration` of more than 3 seconds, that is set to `autoplay` and or `loop`, and has audio output that is not `paused` or `muted`.
```

Comment:
The applicability is not particularly clear and leaves a lot of things open to interpretation. What is `src` file content? What is a `duration`? What does it mean for something to be set to `autoplay` or `loop`? What does it mean for audio to be `paused` or `muted`?

Reply:
I think we can state that these are attributes and link to definitions to clarify most of what is meant.
[duration](https://www.w3.org/TR/html52/semantics-embedded-content.html#dom-htmlmediaelement-duration)
[autoplay](https://www.w3.org/TR/html52/semantics-embedded-content.html#dom-htmlmediaelement-autoplay)
[loop](https://www.w3.org/TR/html52/semantics-embedded-content.html#dom-htmlmediaelement-loop)
[paused](https://www.w3.org/TR/html52/semantics-embedded-content.html#dom-htmlmediaelement-paused)
[muted](https://www.w3.org/TR/html52/semantics-embedded-content.html#dom-htmlmediaelement-muted)

Still, I agree that this can be improved. Here is my attempt
This rule applies to any HTML `<audio>`, `<video>` or `<source>` elements with a `src` attribute referencing auditory content with a `duration` of more than 3 seconds or with a `loop` attribute equal to `true`, that has an `autoplay` attribute equal to true and has attributes `paused` and `muted` equal to false.